gcivil-nyu-org/wed-fall24-team1
# #106 providers can make announcements about their service

# How can you test this?

1. Log in a service provider
2. Either create a service or edit an existing service (make sure you have a user account with this service bookmarked) and add an announcement.
3. The announcement is visible on the services grid in the service dashboard
4. When a regular user searches for this service, finds it and selects the detailed view, the announcement is visible
5. The service status will NOT change (this is intended)
6. Log in as a regular user who has this service bookmarked. You'll get a notification about the announcement
7. The announcement is also visible in the bookmarks tab in the user profile (both normal and detailed view)
